cardano-cli/cardano-cli.cabal

```diff
@@ -108,6 +108,7 @@ library
                        -Wredundant-constraints
                        -Wpartial-fields
                        -Wcompat
+                       -Wno-all-missed-specialisations
```

Comment:
Do we even need that anywhere? We did when we were using `-Weverything` but we dropped than in favour of `-Wall` some time ago. Also, doesn't `cabal build` from the top level ignore these warning flags in favour of the ones in `cabal.project`

Reply:
I don't know if we do need it. cabal does not ignore anything, that's stack.